package.json

```diff
@@ -40,6 +40,7 @@
     "test": "jest --verbose",
     "release:patch": "release patch && npm publish",
     "release:minor": "release minor && npm publish",
-    "release:major": "release major && npm publish"
+    "release:major": "release major && npm publish",
+    "release:pre": "release pre canary && npm publish --tag=canary"
```

Comment:
Copied this over from `nextjs-scripts`, just so I could test a pre-release in advance.